api/envoy/data/orca/v1/BUILD

```diff
@@ -0,0 +1,16 @@
+load("@envoy_api//bazel:api_build_system.bzl", "api_go_proto_library", "api_proto_library_internal")
```

Comment:
I reckon this can be in `api/udpa/data/orca/v1`.

Reply:
ACK.